docs/framework/network-programming/tls.md

```diff
@@ -123,7 +123,7 @@ The WCF framework automatically chooses the highest protocol available up to TLS
 
 We recommend you upgrade your app to .NET Framework 4.7 or later versions. If you cannot upgrade, take the following steps. At some point in the future, your application may fail until you upgrade to .NET Framework 4.7 or later versions.
 
-Set the [SchUseStrongCrypto](#schusestrongcrypto) and [SystemDefaultTlsVersions]#SystemDefaultTlsVersions) registry keys to 1. See [Configuring security via the Windows Registry](#configuring-security-via-the-windows-registry). The .NET Framework version 3.5 supports the `SchUseStrongCrypto` flag only when an explicit TLS value is passed.
+Set the [SchUseStrongCrypto](#schusestrongcrypto) and [SystemDefaultTlsVersions](#SystemDefaultTlsVersions) registry keys to 1. See [Configuring security via the Windows Registry](#configuring-security-via-the-windows-registry). The .NET Framework version 3.5 supports the `SchUseStrongCrypto` flag only when an explicit TLS value is passed.
```

Comment:
I believe the link needs to be lowercase to work, i.e. `#systemdefaulttlsversions`, just like the `#schusestrongcrypto` one just before it.

Reply:
Correct. Line 307 has the right link to that section. I'll fix it right now so we can merge and push it live tonight. Good catch @mderriey!